### PR TITLE
s390x: Fix console login handling

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: lib/susedistribution.pm [hunk 1]
```diff
@@ -384,6 +384,8 @@ sub activate_console {
             sleep 1;
         }
         else {
+            # on s390x we need to login here by providing a password
+            handle_password_prompt if check_var('BACKEND', 's390x');
```
Comment (Soulofdestiny) [inline lib/susedistribution.pm]: not sure about this `if` here

**Comment by Soulofdestiny** [comment]
Just a draft so far
@coolo

**Comment by mnowaksuse** [comment]
>  @mnowaksuse (You are not using the svirt-console, right?)

I don't write much to it apart from `svirt_upload_assets.pm`, but this PR should not interfere.

File: lib/susedistribution.pm [hunk 2]
```diff
@@ -430,14 +432,17 @@ sub activate_console {
     elsif ($type eq 'virtio-terminal') {
         serial_terminal::login($user);
     }
-    elsif ($type eq 'ssh' || $console eq 'svirt') {
+    elsif ($type eq 'ssh') {
         $user ||= 'root';
         assert_screen 'password-prompt';
```
Comment (coolo) [inline lib/susedistribution.pm]: can you make that handle_password_prompt then too? I didn't know about the existance of this

Comment (Soulofdestiny) [inline lib/susedistribution.pm]: updated, though not tested

File: lib/susedistribution.pm [hunk 3]
```diff
@@ -314,6 +314,7 @@ sub init_consoles {
                 'vnc-base',
                 {
                     hostname => $hostname,
+://w3.suse.de/~mgriessmeier/sle-11-s390.webmse testapi;    # FIXME DEBUG
```
Comment (okurz) [inline lib/susedistribution.pm]: remove FIXME debug

**Pull request merged**